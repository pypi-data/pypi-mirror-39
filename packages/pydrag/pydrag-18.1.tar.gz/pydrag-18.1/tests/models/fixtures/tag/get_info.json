{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&lang=en&method=tag.getInfo&tag=rap"
            },
            "response": {
                "body": {
                    "string": "{\"tag\":{\"name\":\"rap\",\"total\":541335,\"reach\":100740,\"wiki\":{\"summary\":\"Rap is a vocal style, usually coming together with hip-hop, the musical genre off-shoot of the hip hop culture. Rapping itself, also known as emceeing, MCing, spitting, or just rhyming, is the rhythmic spoken delivery of rhymes and wordplay. Rapping is one of the four pillars of the hip hop culture, along with DJing, graffiti, and breaking.\\n\\nRap is also considered a separate genre from hip hop in some cases where the artists do not make music compatible with the hip hop culture. <a href=\\\"http://www.last.fm/tag/rap\\\">Read more on Last.fm</a>.\",\"content\":\"Rap is a vocal style, usually coming together with hip-hop, the musical genre off-shoot of the hip hop culture. Rapping itself, also known as emceeing, MCing, spitting, or just rhyming, is the rhythmic spoken delivery of rhymes and wordplay. Rapping is one of the four pillars of the hip hop culture, along with DJing, graffiti, and breaking.\\n\\nRap is also considered a separate genre from hip hop in some cases where the artists do not make music compatible with the hip hop culture. Some of these cases include Lil Wayne, Juelz Santana, Lil Jon, 50 Cent, T.I., The Game, and Nelly. Rap music has a general focus on pop, hyphy, and snap beats, while hip hop has a general focus on the other four pillars of hip hop. <a href=\\\"http://www.last.fm/tag/rap\\\">Read more on Last.fm</a>. User-contributed text is available under the Creative Commons By-SA License; additional terms may apply.\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "1524"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:53 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
