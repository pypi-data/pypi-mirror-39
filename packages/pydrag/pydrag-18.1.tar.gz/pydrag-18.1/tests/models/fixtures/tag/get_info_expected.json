{
    "name": "rap",
    "reach": 100740,
    "total": 541335,
    "wiki": {
        "content": "Rap is a vocal style, usually coming together with hip-hop, the musical genre off-shoot of the hip hop culture. Rapping itself, also known as emceeing, MCing, spitting, or just rhyming, is the rhythmic spoken delivery of rhymes and wordplay. Rapping is one of the four pillars of the hip hop culture, along with DJing, graffiti, and breaking.\n\nRap is also considered a separate genre from hip hop in some cases where the artists do not make music compatible with the hip hop culture. Some of these cases include Lil Wayne, Juelz Santana, Lil Jon, 50 Cent, T.I., The Game, and Nelly. Rap music has a general focus on pop, hyphy, and snap beats, while hip hop has a general focus on the other four pillars of hip hop. <a href=\"http://www.last.fm/tag/rap\">Read more on Last.fm</a>. User-contributed text is available under the Creative Commons By-SA License; additional terms may apply.",
        "summary": "Rap is a vocal style, usually coming together with hip-hop, the musical genre off-shoot of the hip hop culture. Rapping itself, also known as emceeing, MCing, spitting, or just rhyming, is the rhythmic spoken delivery of rhymes and wordplay. Rapping is one of the four pillars of the hip hop culture, along with DJing, graffiti, and breaking.\n\nRap is also considered a separate genre from hip hop in some cases where the artists do not make music compatible with the hip hop culture. <a href=\"http://www.last.fm/tag/rap\">Read more on Last.fm</a>."
    }
}
